geoalchemy2/comparator.py

```diff
@@ -52,6 +53,8 @@
     from sqlalchemy.sql.expression import _FunctionGenerator
 
 
+SA_VERSION = [int(i) for i in SA_VERSION.split('.')[:3]]
```

Comment:
Wouldn't that break with beta versions? For example: https://pypi.org/project/SQLAlchemy/1.3.0b3/.

Reply:
Yes sorry, I was wondering if I should introduce version condition in the code so I chose the dirty way as a first try.

Reply:
No problem. I questioning the introduction of SQLAlchemy version-dependent code anyway.